lib/engine/game/g_1856.rb

```diff
@@ -236,6 +276,27 @@ def setup
         # 1 of each right is reserved w/ the private when it gets bought in. This leaves 2 extra to sell.
         @available_bridge_tokens = 2
         @available_tunnel_tokens = 2
+
+        create_destinations DESTINATIONS
+      end
+
+      def create_destinations(destinations)
```

Comment:
why is this created here? seems like this could be more generic

Reply:
This now exists solely to add destination help text to the corporation charters and to add the icons to the map based on which set of destinations is passed to it. This might also change somewhat later when I work on testing for connectivity, especially in the alternate destination case of the TGB which connects against a 2-hex offboard